src/components/ProgressButton/ProgressButton.jsx

```diff
@@ -13,25 +13,36 @@ class ProgressButton extends React.Component {
 
   render() {
     const beforeText = this.props.beforeText ? (
-      <span className="button-icon">{this.props.beforeText} </span>
+      <span className="button-icon" aria-hidden="true">
+        {this.props.beforeText}
+        &nbsp;
+      </span>
     ) : (
       ''
     );
     const afterText = this.props.afterText ? (
-      <span className="button-icon"> {this.props.afterText}</span>
+      <span className="button-icon" aria-hidden="true">
+        &nbsp;
+        {this.props.afterText}
+      </span>
     ) : (
       ''
     );
 
+    const className =
+      [this.props.buttonClass, this.props.disabled ? 'usa-button-disabled' : '']
+        .filter(Boolean)
+        .join(' ') || null;
```

Comment:
This gets rid of the "undefined" & "null" class name seen in the screenshots.

Reply:
Could we use the `classnames` utility here? Something like:

```js
const className = classnames(this.props.buttonClass, { 'usa-button-disabled': this.props.disabled});
```

Reply:
We could if you want, I don't think we need a dependency for something this simple.